tests/test_feature_extraction_common.py

```diff
@@ -0,0 +1,284 @@
+# coding=utf-8
+# Copyright 2019 HuggingFace Inc.
```

Comment:
The copyrights should all be 2021, they're all at 2020 except this one which is at 2019